cmd/webhook/app/handler.go

```diff
@@ -0,0 +1,67 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+type GateFunc func() (enabled bool)
+
+var (
+	// HandlerMap contains all admission webhook handlers.
+	HandlerMap   = map[string]common.AdmissionHandler{}
+	handlerGates = map[string]GateFunc{}
+)
+
+func addHandlers(m map[string]common.AdmissionHandler) {
+	addHandlersWithGate(m, nil)
+}
+
+func addHandlersWithGate(m map[string]common.AdmissionHandler, fn GateFunc) {
+	for path, handler := range m {
+		if len(path) == 0 {
+			setupLog.Info("Skip handler with empty path.", "handler", handler)
+			continue
+		}
+		if path[0] != '/' {
+			path = "/" + path
+		}
+		_, found := HandlerMap[path]
+		if found {
+			setupLog.Info("conflicting webhook builder path in handler map", "path", path)
```

Comment:
If there is a conflict, only info log?

Reply:
Currently in zap log, we only have info and error level. Do you have any good suggestion?